.mvn/wrapper/maven-wrapper.properties

```diff
@@ -0,0 +1 @@
+distributionUrl=https://repo1.maven.org/maven2/org/apache/maven/apache-maven/3.5.0/apache-maven-3.5.0-bin.zip
```

Comment:
Yikes, it's unfortunate that we'd have to duplicate the version of Maven across each of our repos.  Is it possible to pull this from elsewhere, like the parent POM?